.rubocop_todo.yml

```diff
@@ -0,0 +1,33 @@
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+
+# Offense count: 231
+# Cop supports --auto-correct.
+Style/SpaceInsideParens:
+  Exclude:
+    - 'lib/elastomer/client.rb'
+    - 'lib/elastomer/client/bulk.rb'
+    - 'lib/elastomer/client/cluster.rb'
+    - 'lib/elastomer/client/docs.rb'
+    - 'lib/elastomer/client/errors.rb'
+    - 'lib/elastomer/client/index.rb'
+    - 'lib/elastomer/client/nodes.rb'
+    - 'lib/elastomer/client/scroller.rb'
+    - 'lib/elastomer/client/snapshot.rb'
+    - 'lib/elastomer/client/template.rb'
+    - 'lib/elastomer/middleware/opaque_id.rb'
+    - 'lib/elastomer/notifications.rb'
+    - 'test/client/docs_test.rb'
+
+# Offense count: 1372
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+Style/StringLiterals:
+  Exclude:
+    - '**/*'
```

Comment:
There are so many single quote offenses that rubocop's config generation simply turned it off with `Enable: false` rather than ignoring each individual file. However, that same property was overridden in `.rubocop.yml` to get turned back on again `Enable: true`, so I instead set `Exclude: -'**/*'` to be consistent with the other cops.